pkg/commands/domain/create.go

```diff
@@ -8,17 +8,20 @@ import (
 	"github.com/fastly/cli/pkg/errors"
 	"github.com/fastly/cli/pkg/manifest"
 	"github.com/fastly/cli/pkg/text"
-	"github.com/fastly/go-fastly/v6/fastly"
+	"github.com/fastly/go-fastly/v7/fastly"
 )
 
 // CreateCommand calls the Fastly API to create domains.
 type CreateCommand struct {
 	cmd.Base
-	manifest       manifest.Data
-	Input          fastly.CreateDomainInput
-	serviceName    cmd.OptionalServiceNameID
-	serviceVersion cmd.OptionalServiceVersion
-	autoClone      cmd.OptionalAutoClone
+	manifest manifest.Data
+	// required
+	name           string
+	ServiceName    cmd.OptionalServiceNameID
+	ServiceVersion cmd.OptionalServiceVersion
+	// optional
```

Comment:
nit: I noticed we're a bit inconsistent in spacing in between the required and optional flags, for example elsewhere I've see you include a space:

```
// required
...

// optional
...
```

Reply:
Ah oops- I think most of them have spaces so I'll add the spaces back.

Reply:
Done in https://github.com/fastly/cli/pull/707/commits/a8210d4bb2b90b9427b452f229817faa14326835